packages/public/server/migrations/20201020090526-update-subjects.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * This file is part of Serlo.org.
+ *
+ * Copyright (c) 2013-2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org for the canonical source repository
+ */
+
+/**
+ * Removes some unused subjects and creates new subjects
+ */
+exports.up = function (db, cb) {
+  removeSubject(
+    db,
+    { instance: 1, name: 'Betriebswirtschaftslehre mit Rechnungswesen' },
+    (err) => {
+      if (err) {
+        return cb(err)
+      }
+
+      removeSubject(
+        db,
+        { instance: 1, name: 'Deutsch als Fremdsprache' },
+        (err) => {
+          if (err) {
+            return cb(err)
+          }
+
+          removeSubject(db, { instance: 1, name: 'Musik' }, (err) => {
+            if (err) {
+              return cb(err)
+            }
+            createSubject(db, { instance: 1, name: 'Lerntipps' }, cb)
+          })
+        }
+      )
+    }
+  )
+}
```

Comment:
We might want to write some helpers for that / add a TS toolchain. Code gets quite verbose now.

Reply:
Seems like a Monad-structure to me :smile: